src/Compilers/Core/Portable/SourceGeneration/Nodes/SyntaxValueProvider_ForAttributeWithSimpleName.cs

```diff
@@ -103,13 +107,10 @@ public partial struct SyntaxValueProvider
             .Combine(allUpGlobalAliasesProvider)
             .WithTrackingName("compilationUnitAndGlobalAliases_ForAttribute");
 
-        // For each pair of compilation unit + global aliases, walk the compilation unit 
-        var result = syntaxTreeAndGlobalAliasesProvider
-            .SelectMany((globalAliasesAndCompilationUnit, cancellationToken) => GetMatchingNodes(
-                syntaxHelper, globalAliasesAndCompilationUnit.Right, globalAliasesAndCompilationUnit.Left, simpleName, predicate, cancellationToken))
-            .WithTrackingName("result_ForAttribute");
-
-        return result;
+        return syntaxTreeAndGlobalAliasesProvider.Select(
+            (tuple, c) => (tuple.Left, GetMatchingNodes(syntaxHelper, tuple.Right, tuple.Left, simpleName, predicate, c)))
```

Comment:
```suggestion
        return syntaxTreeAndGlobalAliasesProvider
            .Select((tuple, c) => (tuple.Left, GetMatchingNodes(syntaxHelper, tuple.Right, tuple.Left, simpleName, predicate, c)))
```

Seems more style consistent with rest of file

Reply:
Done in 5fc0481b3654137e197098c2d13b53a7b554f0f2